autosklearn/pipeline/components/data_preprocessing/text_encoding/bag_of_word_encoding.py

```diff
@@ -0,0 +1,102 @@
+from typing import Dict, Optional, Tuple, Union
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+import ConfigSpace.hyperparameters as CSH
+from ConfigSpace import EqualsCondition
+
+import numpy as np
+import pandas as pd
+
+from autosklearn.pipeline.base import DATASET_PROPERTIES_TYPE, PIPELINE_DATA_DTYPE
+from autosklearn.pipeline.components.base import AutoSklearnPreprocessingAlgorithm
+from autosklearn.pipeline.constants import DENSE, SPARSE, UNSIGNED_DATA, INPUT
+
+from sklearn.feature_extraction.text import CountVectorizer
+
+
+class BagOfWordEncoder(AutoSklearnPreprocessingAlgorithm):
+    def __init__(
+        self,
+        min_df_choice: str = None,
```

Comment:
It would be great if you do `typing.Optional[str]` to make it more obvious that this also accepts `None`.